packages/components/src/components/brandshape/brandshape.stories.ts

```diff
@@ -35,157 +42,90 @@ export default {
   }
 };
 
-const increaseColumnWidth = (): ConstantDefinition => {
-  return {
-    type: 'template',
-    name: 'width',
-    value: `<div style="min-width: 300px; max-width: 600px; width: 100vw;">%TEMPLATE%</div>`
-  };
-};
-
 /**
  * Default: This shows sd-brandshape in its default state.
  */
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- *  The different shapes of the brandshape.
+ * Use the `variant` attribute to display the brandshape with different background colors.
  */
-export const Shapes = {
-  name: 'Shapes',
-  parameters: { controls: { exclude: ['shapes'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'attribute',
-          name: 'shapes',
-          values: ['["top", "middle", "bottom"]', '["top", "middle"]', '["middle", "bottom"]', '["top"]']
-        }
-      },
-      args,
-      constants: increaseColumnWidth()
-    });
-  }
-};
 
-/**
- *  The different variants of the brandshape.
- */
-export const Variants = {
-  name: 'Variants',
-  parameters: { controls: { exclude: ['variant'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'attribute',
-          name: 'variant',
-          values: ['neutral-100', 'primary', 'white', 'border-primary', 'border-white', 'image']
-        }
-      },
-      options: {
-        templateBackgrounds: {
-          alternate: 'y',
-          colors: [
-            'white',
-            'white',
-            'rgb(var(--sd-color-primary, 0 53 142))',
-            'white',
-            'rgb(var(--sd-color-primary, 0 53 142))',
-            'white'
-          ]
-        }
-      },
-      args,
-      constants: increaseColumnWidth()
-    });
-  }
-};
+export const Variant = {
+  name: 'Variant',
+  render: () => html`
+    <div class="mb-5">
+      <sd-brandshape variant="primary">
+        <p class="sd-headline sd-headline--lg text-white">Primary Variant Example</p>
```

Comment:
@coraliefeil @smfonseca What do you think about using the claim "We work for your investment" on all brand shapes, maybe even in bold and bigger? Or do we have other examples? Maybe we could even show, that the big brand shape would have more content and the others less?

![CleanShot 2024-08-06 at 11 41 23@2x](https://github.com/user-attachments/assets/8d8a3781-6e0d-4e02-93a7-475717932962)